devices/flash-nrf9160/flashdrv.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * NRF9160 Flash driver
+ *
+ * Copyright 2020 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+/* Copy of flash-stm32 code */
+
+#include <hal/hal.h>
+#include <lib/errno.h>
+#include <devices/devs.h>
+
+/* number of flash devices */
+#define FLASH_NO 1
```

Comment:
Is there possibility for more than one flash? If not there's no need for this param

Reply:
Why not `return minor == 0;` then ?

Reply:
Ok, maybe You are right and this is not necessary, I think i will remove it then